pontoon/base/static/js/fluent_interface.js

```diff
@@ -23,6 +23,57 @@ var Pontoon = (function (my) {
       );
   }
 
+  function renderOriginalVariant(value, elements) {
+    return '<li>' +
+      '<span class="id">' +
+        value +
+      '</span>' +
+      '<span class="value">' +
+        Pontoon.fluent.serializePlaceables(elements, true) +
+      '</span>' +
+    '</li>';
+  }
+
+  function renderOriginalSimpleElement(simpleElements, title) {
+    if (simpleElements.length) {
+      var response = renderOriginalVariant(title, simpleElements);
+      simpleElements = [];
+      return response;
+    }
+
+    return '';
+  }
+
+  /*
+   * Render original string elements.
+   */
+  function renderOriginalElements(elements, title) {
+    var content = '';
+    var simpleElements = [];
+
+    elements.forEach(function (element, i, array) {
+      // Adjoining simple elements are concatenated and presented as a simple string
+      if (Pontoon.fluent.isSimpleElement(element)) {
+        simpleElements.push(element);
+
+        if (i === array.length - 1) {
+          content += renderOriginalSimpleElement(simpleElements, title);
+        }
```

Comment:
This won't happen if the last element of `elements` isn't a simple element. Is that what you intended? If so, I would love to see a comment that explains this logic. :)

Reply:
Good point. Related to your comment above, I think we should be better off without `renderOriginalSimpleElement()`.